canonical-web-and-design/latest-news
# Add null checks and warnings for missing containers

## Done
Add null checks and warnings for missing containers

## QA
- Modify the `articlesContainerSelector` value in the index.html
- Refresh and check you get a warning instead of an error in the console.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/latest-news/issues/21
